modules/java/pages/_partials/gs-additional-steps-for-linux.adoc

```diff
@@ -14,13 +14,15 @@ NOTE: These libraries are provided only in the `.zip` distributable.
 Web Service/Tomcat::
 +
 --
-. Copy the _native libraries_ (`<pathToCbl>/support`) to a location accessible to -- and usable by -- your build and runtime environments.
-. Add the following libraries to the `LD_LIBRARY_PATH` in `$CATALINA_BASE/bin/setenv.sh:
+. Within the `.zip` file, locate the support files appropriate for your environment.
+. Copy the directory `<pathToCbl>/support` to a location accessible and executable by your build and runtime environments.
```

Comment:
I think that, instead of "pathToCbl", here, it should say \<distribution zip\>  or, perhaps \<exploded distribution zip\>

Reply:
Fixed